tests/test_custom_models.py

```diff
@@ -1420,7 +1443,7 @@ def test_multiple_adapters_automatic_modules_to_save(self):
         assert "default" in model.base_model.classifier.modules_to_save
         assert "other" in model.base_model.classifier.modules_to_save
 
-    @parameterized.expand([IA3Config, LoHaConfig, LoKrConfig, LoraConfig, HRAConfig])
+    @parameterized.expand([IA3Config, LoHaConfig, LoKrConfig, LoraConfig, HRAConfig, BoneConfig(r=2)])
```

Comment:
This does not quite work, we need the class here, not an instance:
```suggestion
    @parameterized.expand([IA3Config, LoHaConfig, LoKrConfig, LoraConfig, HRAConfig, BoneConfig])
```

Instead, below, add these lines:

```python
        if config_cls == BoneConfig:
            config_cls = partial(config_cls, r=2)
```

There is already similar code for IA3, just put this code blow that.